src/components/App.js

```diff
@@ -1,15 +1,15 @@
-import React, { Component } from "react";
+import React, { Component } from 'react';
```

Comment:
Nice work, but the convention we are going to use is placing the **stateful** components e.g `App.js` component in the **containers** folder, and placing the **stateless**. So, in this case, shift this component in the container folder and leave this for stateless components. 

When you resolve all these issues let me know.